test/Main.hs

```diff
@@ -1,10 +1,25 @@
+{-# LANGUAGE OverloadedStrings #-}
+
 module Main (main) where
 
+import Data.Text (pack, replace, unpack)
+import System.Exit
+
 import Test.Hspec.Runner
 import Test.Hspec.Formatters.Codewars (codewars)
```

Comment:
We can export `escapeLF` (maybe improve the name) and import it here.

Reply:
I do not know :(

Do you mean to create a new module (i.e. a `*.hs` file), move the function there, and reuse it in two places?

Reply:
Export it from `codewars.hs`
```haskell
module Test.Hspec.Formatters.Codewars (codewars, escapeLF) where
```
then import it here

```haskell
import Test.Hspec.Formatters.Codewars (codewars, escapeLF)
```

Reply:
`LF` in `escapeLF` looks kind of awkward, so maybe rename to `escapeLf` or `escapeLn`?